packages/core/mesh/teleport/src/muxing/framer.ts

```diff
@@ -18,6 +27,9 @@ export class Framer {
   private _messageCb?: (msg: Uint8Array) => void;
   private _subscribeCb?: () => void;
   private _buffer?: Buffer; // The rest of the bytes from the previous write call.
+
+  // TODO(egorgripasov): Will cause a memory leak if streams do not appreciate the backpressure.
+  private _chunkQueue: Chunk[] = [];
```

Comment:
Add a warning for when the queued up chunks surpass a certain theshold (in bytes)